parameters.py

```diff
@@ -0,0 +1,31 @@
+"""
+This file contains all constant test parameters which may be altered from this single
+location for convenience.
+"""
+
+class Parameters:
+
```

Comment:
Add a class description with short description of purpose, and some suggested parameters to set in this class object.